bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -414,13 +414,14 @@ public Closeable createClosable(final Channel channel) {
           public void close() throws IOException {
             ChannelImpl channelImpl = (ChannelImpl) channel;
             channelImpl.shutdown();
-            while (!channelImpl.isTerminated()) {
-              try {
-                channelImpl.awaitTermination(CHANNEL_TERMINATE_WAIT_MS, TimeUnit.MILLISECONDS);
-              } catch (InterruptedException e) {
-                Thread.interrupted();
-                throw new IOException("Interrupted while sleeping for close", e);
-              }
+            try {
+              channelImpl.awaitTermination(CHANNEL_TERMINATE_WAIT_MS, TimeUnit.MILLISECONDS);
```

Comment:
We need to stop at some point.  10 seconds seems fair.  We shouldn't continue indefinitely.  Do you have a better suggestion?